src/doc/trpl/standard-input.md

```diff
@@ -61,11 +67,15 @@ fn main() {
 Next up:
 
 ```{rust,ignore}
-.read_line()
+.read_line(&mut input)
 ```
 
-The `read_line()` method can be called on the result of `stdin()` to return
-a full line of input. Nice and easy.
+The `read_line()` method can be called on the result of `stdin()` to write
+a full line of input to its argument, a String buffer. We pass `input` as a
+mutable reference, since `read_line()` will be modifying the buffer. Every
```

Comment:
References aren't mentioned until chapter 3.3 of the book. So it might be nice to say something like you did on L118, so newbies (like myself) don't get confused.